formstack-baton-requests/src/main/scala/com/gu/identity/formstackbatonrequests/services/FormstackService.scala

```diff
@@ -69,7 +69,8 @@ object FormstackService extends FormstackRequestService with LazyLogging {
 
     /* There are a couple of forms that the Formstack API can't seem to decrypt. There seems to be no way around this
      *  so we capture this specific error and skip these forms. */
-    if(response.body.contains("An error occurred while decrypting the submissions"))
+    // We also skip responses from Formstack that contain "Incorrect password". These are undocumented, and we do not know why they happen.
```

Comment:
Can we identify which forms these are and `WARN` log the ID & reason here (same for the decryption errors)?

I think this is an ok fix, so long as these auth errors are happening extremely infrequently.